### PR TITLE
Covering support case about etag propagation

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Checking if etags are propagated correctly.

### CODE REVIEW

**Comment by SergioBertolinSG** [comment]
@PVince81 is the problem from https://github.com/owncloud/enterprise/issues/1961 well covered with this test?

**Comment by PVince81** [comment]
Apparently not as it doesn't seem to reproduce the issue ?

**Comment by SergioBertolinSG** [comment]
From the issue I understood the steps were something like:

User A shares remotely shared_folder with user B.
User A modifies something in shared_folder.
Etags were propagated correctly

Should they be different?

File: tests/integration/federation_features/federated.feature [hunk 1]
```diff
@@ -191,6 +191,21 @@ Feature: federated
...
+		And user "user1" exists
+		And Using server "LOCAL"
+		And user "user0" exists
+		And User "user0" from server "LOCAL" shares "/PARENT" with user "user1" from server "REMOTE"
+		And User "user1" from server "REMOTE" accepts last pending share
+		And Using server "REMOTE"
+		And user "user1" stores etag of element "/PARENT (2)"
+		And Using server "LOCAL"
+		And As an "user0"
+		When User "user0" uploads file "data/file_to_overwrite.txt" to "/PARENT/textfile0.txt"
```
Comment (PVince81) [inline tests/integration/federation_features/federated.feature]: write another test where user1 uploads  a file and the etag for user0 changed

Comment (SergioBertolinSG) [inline tests/integration/federation_features/federated.feature]: Added, but this case is passing as well.

**Comment by PVince81** [comment]
Then let's merge them and backport to stable9.1 and stable9. The bug was reported on 9.0.8

**Comment by PVince81** [comment]
:+1:

**Comment by SergioBertolinSG** [comment]
OK I'll prepare the backports.

**Comment by SergioBertolinSG** [comment]
Backports:
stable9    https://github.com/owncloud/core/pull/27802
stable9.1 https://github.com/owncloud/core/issues/27801

In stable9 cherry-pick failed so I just created a new commit.

**Pull request merged**